capellambse/model/crosslayer/information.py

```diff
@@ -52,17 +52,102 @@ class Class(c.GenericElement):
 
     _xmltag = "ownedClasses"
 
-    inheritance = c.ProxyAccessor(capellacore.Generalization)
+    sub = c.Accessor
+    super = c.Accessor
     state_machines = c.ProxyAccessor(
         capellacommon.StateMachine, aslist=c.ElementList
     )
 
 
+@c.xtype_handler(None)
+class Union(c.GenericElement):
+    """A Union."""
+
+    _xmltag = "ownedClasses"
+
+    state_machines = c.ProxyAccessor(
+        capellacommon.StateMachine, aslist=c.ElementList
+    )
+
+
+@c.xtype_handler(None)
+class Collection(c.GenericElement):
+    """A Collection."""
+
+    _xmltag = "ownedCollections"
+
+    state_machines = c.ProxyAccessor(
+        capellacommon.StateMachine, aslist=c.ElementList
+    )
+
+
+@c.xtype_handler(
+    None, "org.polarsys.capella.core.data.information.datavalue:ComplexValue"
+)
+class ComplexValue(c.GenericElement):
+    """A Complex Value."""
+
+    _xmltag = "ownedDataValues"
+    type = c.AttrProxyAccessor(c.GenericElement, "abstractType")
+
+
+@c.xtype_handler(
+    None,
+    "org.polarsys.capella.core.data.information.datavalue:EnumerationLiteral",
+)
+class EnumerationLiteral(c.GenericElement):
+    """An EnuemrationLiteral (proxy link)."""
+
+    _xmltag = "ownedLiterals"
+
+    owner: c.Accessor
+
+
+@c.xtype_handler(
+    None, "org.polarsys.capella.core.data.information.datatype:Enumeration"
+)
+class Enumeration(c.GenericElement):
+    """An Enumeration."""
+
+    _xmltag = "ownedDataTypes"
+
+    sub = c.Accessor
+    super: Enumeration = c.Accessor  # type: ignore
+    own_literals = c.ProxyAccessor(
+        EnumerationLiteral,
+        "org.polarsys.capella.core.data.information.datavalue:EnumerationLiteral",
+        aslist=c.ElementList,
+        follow_abstract=False,
+    )
+
+    @property
+    def literals(self) -> c.ElementList[EnumerationLiteral]:
+        """returns own + inherited literals"""
```

Comment:
Docstring formatting, please use imperative style and make this a sentence (just like the other function/method docstrings) :)

```suggestion
        """Return all owned and inherited literals."""
```